biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/BndPomRepository.java

```diff
@@ -151,7 +181,10 @@ private void poll() throws Exception {
 
 	@Override
 	public boolean refresh() throws Exception {
-		init();
+		prepare();
```

Comment:
don't use prepare. Refresh is about the pom file but not about the settings. If setting sin the bnd file change, a new plugin is created